### PR TITLE
Improved README and fixed a few things.

### PR DESCRIPTION
**Pull request by azizk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,21 +135,23 @@ To convert the duration from native units you can use:
...
-    end
-  )
+
+  result =
+    :telemetry.span(
+      [:worker, :processing],
+      start_metadata,
+      fn ->
+        # Process the message
+        result = {result, %{metadata: "Information related to the processing of the message"}}
```
Comment (josevalim) [inline README.md:152]:
```suggestion
        result = ... # Process the message
        {result, %{metadata: "Information related to the processing of the message"}}
```

File: README.md [hunk 2]
```diff
@@ -132,21 +135,23 @@ To convert the duration from native units you can use:
...
-  result = :telemetry.span(
-    [:worker, :processing],
-    start_metadata,
-    fn ->
-      result = # Process the message
-      {result, %{metadata: "Information related to the processing of the message"}}
-    end
-  )
+
+  result =
```
Comment (josevalim) [inline README.md:146]: Should we keep this var?

Comment (azizk) [reply README.md:146]: Maybe not. I just formatted it with the formatter. :)

**Review by josevalim — APPROVED** [review]

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**